src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -84,7 +88,10 @@ export const NormalAnnouncementBubble: React.FC<Props> = ({
       {attachments?.length > 0 && (
         <FlexBox className={cssClass("attachmentContainer")}>{attachments}</FlexBox>
       )}
-      {readReceiptsTooltip}
+      <FlexBox justify="between">
```

Comment:
`justify="between"` along with adding `margin-left: auto;` and deleting `width: 100%` in the .less file ensure that 
1) text divs grow with text
2) extra width between the two tooltips when both are present is distributed evenly
3) that extra space^ is not hoverable